.github/ISSUE_TEMPLATE/usability_test.md

```diff
@@ -0,0 +1,65 @@
+---
+name: Usability testing
+about: Propose a usability test to help us
+title: [UX]
+labels: enhacement
+---
+# Objective
+The test goals. e.g.: Evaluate the account customizations feature and website information placement.
+
+# Requirements
+
+## Environment
+What is the environment that should be tested and how it should be prepared. e.g.: The test needs to run in the pre-2.0 version of the project and a browser with the main page opened.
+
+## User profile
+Describe the target users you are looking for the test. e.g.: A user that has already used other projects similar to Houdini.
+
+## Facilitator
+What the facilitator should be familiar with to run the tests. e.g.: The facilitator needs to understand the user journey to donate to a project.
+
+# Test design
+## Tasks
+A list of tasks that the user has to perform. e.g.:
+
+* Change profile information.
+* Discover the project features.
+
+## Scenarios
+A list of scenarios for the user to perform the tasks. They should put the user in a context and not give specific hints about what you want the user to do. e.g.:
+
+1. You received an e-mail asking to update your personal information because your complete name is incorrect. Please, look for this option in the application.
+2. When you were navigating through the application, you realized how awesome it is and you want to discover what else you can do with it. Can you find out what else the application does?
+
+<!-- 
+
+## Results
+
+Uncomment this session once you have your results.
+
+### Summary
+
+|   -    | User 1 | User 2 | User 3 | User 4 | User 5 |
+|:------:|:------:|:--------:|:------------------:|:------:|:------:|
+| Task 1 | :x:                   | :question: | :heavy_check_mark: | :x: | :heavy_check_mark: |
+| Task 2 | :heavy_check_mark:    | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Task 3 | :heavy_check_mark:    | :heavy_check_mark: | :x: | :heavy_check_mark: | :heavy_check_mark: |
+| Task 4 | :heavy_check_mark:    | :question: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+
+### Task 1
+
+#### What went well?
+Users intuitively clicked on the "?" icon to find the help documentation.
+
+#### What were the challenges?
+Some users didn't understand how to create a new blog post inside a folder, they first created the post and then edited the folder to include the post inside of it. They said they were looking for a select input to choose where the blog post should be placed instead of a folder icon.
+
+### Task 2
+
+#### What went well?
+Users intuitively clicked on the "?" icon to find the help documentation.
+
+#### What were the challenges?
+Some users didn't understand how to create a new blog post inside a folder, they first created the post and then edited the folder to include the post inside of it. They said they were looking for a select input to choose where the blog post should be placed instead of a folder icon.
+
+-->
```

Comment:
The placement for the results is just a suggestion. Some projects may prefer to have the results registered on the wiki or somewhere else. Personally, I like placing in the issues :smiley:

Reply:
I like issues as well!